addons/point_of_sale/static/src/js/models.js

```diff
@@ -1372,6 +1379,14 @@ exports.PosModel = Backbone.Model.extend({
 
     disallowLineQuantityChange() {
         return false;
+    },
+
+    getCurrencySymbol() {
+        return this.currency ? this.currency.symbol : '$';
+    },
+
+    isOrderValidated(order) {
```

Comment:
do you really need this one? as yoou always pass the order, can't you just check order.finalized where you call isOrderValidated

Reply:
I thought of the same too so I guess that's the right move.